src/app/client/src/app/modules/dashboard/components/course-progress/course-progress.component.html

```diff
@@ -134,15 +134,15 @@
               [disabled]="!scoreReportUpdatedOn"
               [telemetryInteractEdata]="{id:'download-score-report',type:'click',pageid:'course-progress-dashboard'}"><i
                 class="download icon"></i>{{resourceService?.frmelmnts?.instn?.t0066}}
-                <span class="ml-8" suiPopup popupText="{{!scoreReportUpdatedOn?'Report is not available':resourceService?.frmelmnts?.lbl?.reportUpdatedOn}} {{scoreReportUpdatedOn | date:'dd/MM/yyyy'}}"[popupPlacement]="'top right'" [popupInverted]="true"><img src="assets/images/info-outline.svg" alt="info-outline" class="info-outline-img"></span>
+                <span class="ml-8" suiPopup popupText="{{!scoreReportUpdatedOn ? resourceService?.messages?.emsg?.m0076 : resourceService?.frmelmnts?.lbl?.reportUpdatedOn}} {{scoreReportUpdatedOn | date:'dd/MM/yyyy'}}"[popupPlacement]="'top right'" [popupInverted]="true"><img src="assets/images/info-outline.svg" alt="info-outline" class="info-outline-img"></span>
```

Comment:
adding normal condition will be more readable and less confusing.
sample code snippet
scoreReportUpdatedOn ? resourceService?.frmelmnts?.lbl?.reportUpdatedOn : resourceService?.messages?.emsg?.m0076